h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -6,11 +6,21 @@ function Star() {
   return <span className="text-brand">*</span>;
 }
 
-function CharacterCounter({ value, limit }: { value: number; limit: number }) {
+function CharacterCounter({
+  value,
+  limit,
+  testid,
+}: {
+  value: number;
+  limit: number;
+  testid: string;
+}) {
   return (
     <div className="flex">
       <div className="grow" />
-      {value}/{limit}
+      <span data-testid={testid}>
+        {value}/{limit}
+      </span>
```

Comment:
We might want to add an `aria-description` here in future to clarify the meaning of these numbers for assistive technology users. Something like ``<span aria-description={`${value} of ${limit} characters used`}>...</span>``.

Reply:
Added a note of this to the project board